src/core/core_structs/chronology.jl

```diff
@@ -1,10 +1,18 @@
 abstract type AbstractChronology end
 
+struct Consecutive <: AbstractChronology end
+
 struct Synchronize <: AbstractChronology
-    from_horizon::Int64 #number of time periods to grab data from
-    to_steps::Int64 #number of times to run using the same data
+    from_steps::Int64    #number of time periods to grab data from
+    to_executions::Int64 #number of times to run using the same data
+    function Synchronize(;from_steps, to_executions)
+        new(from_steps, to_executions)
+    end
 end
 
-struct RecedingHorizon <: AbstractChronology end
-
-struct Sequential <: AbstractChronology end
+struct RecedingHorizon <: AbstractChronology
+    step::Int64
+    function RecedingHorizon(;from_step::Int64=1)
```

Comment:
By specifying the type as an Int you prevent the user from passing a non-Int that could be converted to an Int, like a Float64.  You're probably better off not specifying the type on inner constructor parameters.

Reply:
Do these need to be Int64 rather than just Int?  Whenever I see that I'm assuming that the developer is expecting numbers bigger than Int32 can hold.

Reply:
I think we always use Int64, I don't think it matters too much.